tf-psa-crypto/drivers/builtin/src/ecdsa.c

```diff
@@ -193,8 +193,6 @@ static void ecdsa_restart_det_free(mbedtls_ecdsa_restart_det_ctx *ctx)
 
 #endif /* MBEDTLS_ECP_RESTARTABLE */
 
-#if defined(MBEDTLS_ECDSA_DETERMINISTIC) || \
```

Comment:
We do not keep "#if defined(MBEDTLS_ECDSA_DETERMINISTIC)" here?

Reply:
Surely we don't need to? The following code was needed if `MBEDTLS_ECDSA_SIGN_ALT` isn't defined, or if `MBEDTLS_ECDSA_VERIFY_ALT` isn't defined, or if `MBEDTLS_ECDSA_DETERMINISTIC` was defined. Well, both those first two will always not be defined now

Reply:
Indeed.